src/cucu/environment.py

```diff
@@ -215,6 +215,20 @@ def after_scenario(ctx, scenario):
     for timer_name in ctx.step_timers:
         logger.warning(f'timer "{timer_name}" was never stopped/recorded')
 
+    browser_info = {"has_browser": False}
+
+    if len(ctx.browsers) != 0 and not CONFIG.true("CUCU_KEEP_BROWSER_ALIVE"):
```

Comment:
we're not closing the browser in get tab here
```suggestion
    if len(ctx.browsers) != 0:
```